src/messages/Image.cpp

```diff
@@ -33,8 +38,8 @@ namespace detail {
         DebugCount::increase("images");
     }
 
-    Frames::Frames(const QVector<Frame<QPixmap>> &frames)
-        : items_(frames)
+    Frames::Frames(QVector<Frame<QPixmap>> &&frames)
+        : items_(std::move(frames))
```

Comment:
For reference, QVector is one of the `implicit shared` classes of Qt where the class itself doesn't store the data, but rather essentially a shared_ptr to data, so copying just adds a reference to the underlying data https://doc-snapshots.qt.io/qt5-5.15/implicit-sharing.html
Either way, I'm fine with this change

Reply:
I see, I was wondering why we were using QVector at all